main.go

```diff
@@ -171,15 +173,29 @@ func registerEHR(server *echo.Echo, config Config, customerRepository customers.
 		log.Fatal(err)
 	}
 
-	fhirClientFactory := fhir.NewFactory(fhir.WithURL(config.FHIR.Server.Address), fhir.WithMultiTenancyEnabled(config.FHIR.Server.SupportsMultiTenancy()))
+	fhirNotifier := transfer.FireAndForgetNotifier{}
+	var tlsClientConfig *tls.Config
+	if config.TLS.Client.IsConfigured() {
+		log.Println("Configuring TLS client certificate for calls to remote Nuts Nodes and FHIR servers.")
+		if tlsClientConfig, err = config.TLS.Client.Load(); err != nil {
+			log.Fatal(err)
+		}
+		fhirNotifier.TLSConfig = tlsClientConfig
+	}
+
+	fhirClientFactory := fhir.NewFactory(
+		fhir.WithURL(config.FHIR.Server.Address),
+		fhir.WithMultiTenancyEnabled(config.FHIR.Server.SupportsMultiTenancy()),
+		fhir.WithTLS(tlsClientConfig),
```

Comment:
what happens with a nil pointer here? (eg: no TLS config)

Reply:
sets nil, which is the default value